apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -1089,6 +1104,28 @@ export default Vue.extend({
       pendingUpdate.cell.getElement().classList.remove('edited')
       pendingUpdate.cell.getElement().classList.remove('edit-error')
     },
+    showColumnFilterModal() {
+      this.$modal.show('column-filter-modal', {
+        allColumns: [...this.table.columns],
+        selectedColumnNames: [...this.selectedColumnNames],
+        onApply: (selectedColumnNames) => {
```

Comment:
So instead of providing `allColumns`, `selectedColumns`, and `onApply` to the modal function (which I recognize as the React-way to do this), in Vue you're best to provide properties and callbacks to the component. This will also make it more consistent with the rest of the App. The component itself can filter 


```vue
<ColumnFilterModal :columns="columnsWithFilterAndOrder" @changed="applyColumnChanges" />

```